### ISSUE TITLE
Skyshowtime and rakuten

### ISSUE THREAD

**Issue by omakoti** [Nerwyn/android-tv-card#27]
Hi.
Can you add Skyshowtime and RakutenTV to the list of ready made buttons. I'm bad at coding so don't know how to add them myself.

**Comment by Nerwyn** [comment]
Putting the custom source config here for now to reference later:

```yaml
custom_sources:
  skyshowtime:
    source: 'https://www.skyshowtime.com/deeplink'
    icon: skyshowtime
  rakutenviki:
    source: 'viki://home'
    icon: rakutenviki
custom_icons:
  skyshowtime: 'TODO'
  rakutenviki: 'TODO'
```

**Comment by Nerwyn** [comment]
Released [v3.0.1](https://github.com/Nerwyn/android-tv-card/releases/tag/v3.0.1), which adds `videoland`, `skyshowtime`, and `rakutenviki` as default sources with SVG icons.
